external-servers/dnsmasq/Dockerfile

```diff
@@ -0,0 +1,10 @@
+# This could be a lighter image, but for development use ubuntu 22.04 
+FROM ubuntu:22.04
```

Comment:
Note that I haven't tested this specific image as we can't use Docker Hub images at TI, but this should work.